docs/_data/spinner.json

```diff
@@ -24,6 +24,11 @@
       "class": ".s-spinner__lg",
       "applies": ".s-spinner",
       "description": "A large style for the largest layouts"
+    },
+    {
+      "class": ".is-loading",
+      "applies": "Any element",
```

Comment:
I tried this on a bunch of miscellaneous components and it works _to varying degrees_. If an element already has pseudo-elements, this class is likely to overwrite (or at least append) them. I found this with `.s-select`. In many other cases, the loading spinner isn't quite aligned the way you'd expect.

Anyone applying this class will need to use it thoughtfully and I can see a future where someone comes to the Stacks team in need of a component-specific application. That being said, I think this is progress.

Reply:
Hmm... that's a good call. I'll add some notes to the docs. Instead of "any element" maybe something like "any text display element"? That's pretty confusing though. I'll also drop a note to prefer using the spinner markup if possible.

Alternatively, we could leave this relatively undocumented and just officially support the class on `s-btn`, but I'd like to avoid that scenario if we can.

Reply:
I updated the documentation in https://github.com/StackExchange/Stacks/pull/1040/commits/ac44000f502cdd247d342c7d86bde920fad81528. I'm not 100% satisfied with the wording, but I don't think we need to spend too much more time on it.